src/transfer-manager.ts

```diff
@@ -238,6 +238,8 @@ export class TransferManager {
         prefix: filesOrFolder,
       });
       files = directoryFiles[0];
+      // filter out directories
```

Comment:
```suggestion
      // filter out directory objects
```

Reply:
https://cloud.google.com/knowledge/kb/listing-only-objects-using-gsutil-and-not-directories-000004568

Just for the record
![Screen Shot 2023-05-28 at 6 43 43 PM](https://github.com/googleapis/nodejs-storage/assets/12564636/353cf066-782c-47bb-a7b9-e0d3fa2313c2)